pkg/reporter/summary.go

```diff
@@ -727,6 +737,11 @@ func (r *summaryReporter) provenanceStatement() string {
 		r.summary.provenance.none)
 }
 
+func (r *summaryReporter) depsusageEvidenceStatement() string {
+	return fmt.Sprintf("%d/%d libraries unused",
```

Comment:
We should probably state that `X` libraries has usage evidence among `Y` libraries. I think we are not yet there where we can "factually state" if a library is unused. We are looking for usage evidence so we can only "state" that as truth.

Reply:
ack 👍